share/resources/gflags.json

```diff
@@ -0,0 +1,37 @@
+{
+    "IMMUTABLE": [
```

Comment:
Could we start all services on one host with just one gflags.json?

Reply:
Yeah, all services will try to read this file.